docs/logs/using.asciidoc

```diff
@@ -1,112 +1,86 @@
+Log entries for the time you specify appear in the middle of the page, with the earlier entries above and the later entries below.
 
+To quickly jump to a nearby point in time, click the minimap timeline to the right.
+// ++ what's this thing called? It's minimap in the UI. Would timeline be better?
 
 [float]
 [[logs-customize]]
 === Customize your view
-Use *Customize* to adjust your console view and to set the time scale of the log data.
-
-* *Text size.*  Select `Small`, `Medium`, or `Large`.
-* *Wrap long lines.* Enable or disable line wrap.
-* *Minimap Scale.* Set the scale to 'year', 'month', 'week', 'day', 'hour', or 'minute'.
-
-[role="screenshot"]
-image::logs/images/logs-usage-customize.png[Logs view customization popover]
+Click *Customize* to customize the view.
+Here, you can set the scale to use for the minimap timeline, choose whether to wrap long lines, and choose your preferred text size.
 
 [float]
 [[logs-stream]]
 === Stream or pause logs
-You can stream data for live log tailing, or pause streaming to focus on historical log data.
+Click *Stream live* to start streaming live log data, or click *Stop streaming* to focus on historical data.
 
 [role="screenshot"]
 image::logs/images/logs-usage-start-streaming.png[Logs start streaming]
 
 [role="screenshot"]
 image::logs/images/logs-usage-stop-streaming.png[Logs stop streaming]
 
-When you are streaming logs, the most recent log appears at the bottom on the console.
+When you are viewing historical data, you can scroll back through the entries as far as there is data available.
 
-[role="screenshot"]
-image::logs/images/logs-usage-streaming-indicator.png[Logs streaming indicator]
-
-Historical data offers infinite scrolling.
+When you are streaming live data, the most recent log appears at the bottom of the page.
+In live streaming mode, you are not able to choose a different time in the time selector or use the minimap timeline.
+To do either of these things, you need to stop live streaming first.
+// ++ Not sure whether this is correct or not. And what about just scrolling through the display?
+// ++ There may be a bug here, (I managed to get future logs) see https://github.com/elastic/kibana/issues/43361
 
 [float]
 [[logs-highlight]]
 === Highlight a phrase in the logs stream
-To find instances of a particular word or phrase in the logs stream, click the Highlights button in the toolbar and enter your search terms.
-
-[role="screenshot"]
-image::logs/images/logs-stream-highlight-box.png[Logs highlight dialog box]
-
-This will highlight any instances of your search terms that appear in the logs stream.
-
-[role="screenshot"]
-image::logs/images/logs-stream-highlight-entries.png[Logs stream with the search term 'safepoint' highlighted]
+To highlight a word or phrase in the logs stream, click *Highlights* and enter your search phrase.
+// ++ Is search case sensitive?
+// ++ Can you search for multiple phrases together, if so, what's the separator?
```

Comment:
A comma, for example: 
<img width="434" alt="Screen Shot 2019-08-16 at 9 53 48 AM" src="https://user-images.githubusercontent.com/5618806/63184239-c42d7f00-c00b-11e9-82ab-2b110000495e.png">
"DEBUG","component"

Reply:
That only works if the two words are contiguous (or, it seems, only separated by some kinds of separators).

![image](https://user-images.githubusercontent.com/20440352/63264721-acf8c680-c283-11e9-9b7c-f1e96be9a108.png)

Incidentally, `/` and `:` also seem to work as separators, as does white space, but again, only if the text strings being reached for are contiguous, apart from separators.

@Zacqary I think you implemented this? If so, can you confirm what the expected behaviour should be, please? I'm not sure whether what I'm seeing is the intended behaviour or not.

- is it possible to specify multiple non-contiguous words to highlight, and if so, how?
- some items in the search string seem to be treated as separators, not search items (for example comma, double quotes, forward slash, colon) whereas others are treated as a part of the search string, for example full stop? Is that expected behaviour?
- it seems that it's (usually) only possible to search for and highlight whole words (where there is some variation in what constitutes a "word" and what the word separators are). Is that the case?

For example, one of the lines in my log starts `File /usr/local/lib/python3.6/site-packages/redis/client.py", line 3096, in subscribe . . . `.

Searching for `site` highlights `site-packages`, searching for `python` or `client` highlights nothing, although the text exists in this line, searching for `client.py` highlights `client.py`, searching for `usr local` highlights `usr` and `local` but not the `/` that separates them.

Reply:
@Kerry350 implemented and she can clarify, I just added the docs.

Reply:
The search terms are [analyzed](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis.html) by Elasticsearch and turned into a [phrase query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query-phrase.html). That means that white-space and punctuation are treated the same as in the indexed log entries.

AFAIK there's currently no efficient way to perform exact substring searches in Elasticsearch.